cmd/log.go

```diff
@@ -211,7 +211,7 @@ func (o *LogbookOptions) Complete(f Factory, args []string) (err error) {
 	return err
 }
 
-// Logbook executes the Logbook command
+// LogEntries executes the Logbook command
```

Comment:
I have the same fix in https://github.com/qri-io/qri/pull/1733/files, but wrote "LogEntries gets entries from the logbook" which more closely describes what's going on.

Reply:
That's fine, I'll update mine as well and rebase later given that you will probably merge before I do on this. Sneaked in some other commit I guess.